### ISSUE TITLE
Expose complete user address

### ISSUE THREAD

**Issue by rosano** [fission-suite/webnative#178]
Re: @bmann's [comment](https://talk.fission.codes/t/kommit-fission-integration/1634/5):

> Technically usernames are @fission.name although that isn’t exposed anywhere

perhaps `state.username` could present the domain portion as well. This would allow me to make my funding button work with webnative without hard-coding the domain.

**Comment by bmann** [comment]
Yeah this is a good idea.

I don’t know if it is a username so I think a different variable — federationuser or something like that. 

The portable unique identifier is the DID for the account. The username is a label that we’ve decided to make unique for compatibility / UX reasons.

hmmm. I guess “boris@bmannconsulting.com”  or “boris@fission.name” is the full federation user ID?

**Comment by rosano** [comment]
> hmmm. I guess “boris@bmannconsulting.com” or “boris@fission.name” is the full federation user ID?

I guess if Fission supported Webfinger then the app would probably initiate the process outside of webnative and have one of these on hand already. But until then, either one would would be useful.

**Comment by rosano** [comment]
Perhaps related: if the username/address was encrypted rather than stored in plaintext, the recipient could assume it is the actual identity? Again with my funding system, I can tie things to `state.username` or address without wondering whether someone overwrote it.

**Comment by bmann** [comment]
Yes. Broadly speaking I think we'd follow Verified Credentials for this, which basically links data to an ID.
